FreeRTOS-Plus/Source/FreeRTOS-Plus-CLI/FreeRTOS_CLI.c

```diff
@@ -61,16 +61,18 @@ static BaseType_t prvHelpCommand( char *pcWriteBuffer, size_t xWriteBufferLen, c
 /*
  * Return the number of parameters that follow the command name.
  */
+
 static int8_t prvGetNumberOfParameters( const char *pcCommandString );
 
 /* The definition of the "help" command.  This command is always at the front
 of the list of registered commands. */
 static const CLI_Command_Definition_t xHelpCommand =
 {
 	"help",
-	"\r\nhelp:\r\n Lists all the registered commands\r\n\r\n",
+	"help:\r\n Lists all the registered commands\r\n",
+    "",
 	prvHelpCommand,
-	0
+	-1

```

Comment:
0 means it is expected no parameter while -1 means user can enter any number of parameters. Do you have a reason to change to -1?